flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/MapView.java

```diff
@@ -89,10 +86,12 @@
  *
  * }</pre>
  *
+ * <p>{@code MapView(TypeInformation<?> keyType, TypeInformation<?> valueType)} method was
+ * deprecated and removed. Please use a {@link DataTypeHint} instead.
+ *
  * @param <K> key type
  * @param <V> value type
  */
-@TypeInfo(MapViewTypeInfoFactory.class)
```

Comment:
The same requestion as https://github.com/apache/flink/pull/25565

Reply:
Thanks for reviewing the PR @lsyldliu. I have removed MapViewTypeInfoFactory as it's not used anywhere. MapViewTypeInfo seems to be used in a few places which would need some refactoring. I think it's better to remove them in a separate PR.